datm/cime_config/config_component.xml

```diff
@@ -299,10 +299,10 @@
   <entry id="DATM_YR_START_FILENAME">
     <type>integer</type>
     <valid_values></valid_values>
-    <default_value>UNSET</default_value>
+    <default_value>9999</default_value>
     <group>run_component_datm</group>
     <file>env_run.xml</file>
-    <desc>Start year listed in PLUMBER2 filenames. Leave unset for other cases.</desc>
+    <desc>Start year listed in PLUMBER2 filenames. Leave as 9999 for other cases.</desc>
```

Comment:
I guess I'm a little worried about the name of the variable getting super long, but could replace some other part of the variable name with plumber if necessary. One other note-- if this is later expanded to other towersites, it is possible that it would also be used for non-plumber sites.

Reply:
Sorry, I'm going to muddy the waters here a bit by respectfully disagreeing with some of @ekluzek 's suggestions.

Regarding the default value: I actually prefer 9999 over -9999. If there were a good place to put in a check about the reasonableness of this value, I agree that an obvious non-sensical value would be best. But in this case, the value just gets inserted into a file name pattern, and I feel like inserting a negative value into the file name would be slightly confusing when looking at the resulting file name (because it would put an extra hyphen in the file name). I feel like positive 9999 is still nonsensical enough that, when looking at the file name if you forget to set this, you'll still be able to say, "oh yeah, this got left as uninitialized". It's true that the model occasionally runs out this far, but presumably datm input data like this shouldn't be using year 9999. I don't feel strongly on this at all, but I would cast a weak vote for sticking with 9999.

Regarding the variable name: I also thought about putting PLUMBER2 in the file name. My own feeling is that it's better to keep it generic like this: There's nothing inherent about this that says that it can / should only be used in PLUMBER2, and I could see extending it later to apply to more situations. Putting PLUMBER2 in the name would make it challenging to extend later, because you'd want to rename it at that point, and then existing workflows would need to be changed. So I prefer just keeping the documentation clear.

Given that, maybe a way to write the description would be something like:

```
Start year listed in filenames for certain datm_modes.

Currently only used with PLUMBER2. Leave at the default value for other cases.
```

Reply:
Oops, I see that @TeaganKing already addressed @ekluzek 's comments before I finished making this comment. I don't feel strongly on any of this so I'm fine with just moving ahead with what you have now so you don't have to go back and forth on this!

Reply:
Ok, I think that I addressed this sentiment in the variable description, and could see pros/cons with either the positive or negative value but am happy to leave as it currently is.

Reply:
Thanks @TeaganKing and @billsacks yes I think you are right thanks for explaining the reasoning. That's a good point that the resulting filename would have a double hyphen and be weird to interpret. I like @billsacks suggestion on the description above.

Reply:
Okay, I see that I indeed muddied the waters a lot here. @TeaganKing I'm very happy to let you make the executive decision on how you want to go ahead with @ekluzek 's suggestion!

Reply:
Ok I just made the value positive and expanded the description a bit more. I'll resolve this comment now :)